### PR TITLE
[SYTO-883] Restyle Modal (Mortar)

### PR DESCRIPTION
**Pull request by sebasrgl**
https://jira.natgeo.com/browse/SYTO-883

#### How to test
1. ``gulp serve``
2. Open http://localhost:3000/patterns/
3. Go to Modal section and click on the button
4. Check the modal changes.

### CODE REVIEW

File: app/modules/modals/_styles.scss [hunk 1]
```diff
@@ -4,18 +4,24 @@
...
   cursor: pointer;
   position: absolute;
-  right: $mt3_unit-double;
-  top: $mt3_unit;
+  right: $mt3_unit-double - .125;
+  top: $mt3_unit-double - .125;
+
+  //This media query apply to approve the ticket, since we don't have in mortar
+  // that breakpoint.
+  @media #{'(max-width: 767px)'} {
```
Comment (alexrqs) [inline app/modules/modals/_styles.scss]: why do you need to interpolate plain text? isnt this bp defined?

File: app/modules/modals/_styles.scss [hunk 2]
```diff
@@ -4,18 +4,24 @@
...
+    right: $mt3_unit - .0625;
+    top: $mt3_unit - .0625;
+  }
 }
 
 .mt3_modal-container {
-  background-color: transparentize($mt3_color-black, .15);
+  background-color: $mt3_color-gray-86;
```
Comment (alexrqs) [inline app/modules/modals/_styles.scss]: :+1:

File: app/modules/modals/_styles.scss (hunk 1 above)
Comment (alexrqs) [inline app/modules/modals/_styles.scss]: as the comment says is not, maybe you can use only the regular media query expression?

Comment (alexrqs) [inline app/modules/modals/_styles.scss]:
```css
@media (min-width: 700px) { ... }
```

File: app/modules/modals/_styles.scss [hunk 3]
```diff
@@ -4,18 +4,24 @@
...
   height: 85vh;
   left: 7.5vw;
   opacity: 0;
   position: fixed;
   top: 7.5vh;
-  transition: .2s;
```
Comment (alexrqs) [inline app/modules/modals/_styles.scss]: :open_mouth:

File: app/modules/modals/_styles.scss (hunk 1 above)
Comment (sebasrgl) [inline app/modules/modals/_styles.scss]: Fixed but, not outdiff

Comment (alexrqs) [inline app/modules/modals/_styles.scss]: wtf

**Pull request merged**